### PR TITLE
polymorphic playground

### PR DESCRIPTION
**Pull request by toddpla**

### CODE REVIEW

**Comment by runticle** [comment]
pretty excited about this stuff

File: Gemfile [hunk 1]
```diff
@@ -62,3 +64,5 @@ gem "activesupport", "~> 5.1"
 gem "pry", "~> 0.12.2"
 
 gem "timecop", "~> 0.9.1"
+
+gem "jquery-rails", "~> 4.3"
```
Comment (imogenmisso) [inline Gemfile]: May cause conflicts.. we have also added this gem

File: app/controllers/comments_controller.rb [hunk 2]
```diff
@@ -1,11 +1,13 @@
+require 'pry'
```
Comment (imogenmisso) [inline app/controllers/comments_controller.rb]: Can remove

File: app/controllers/comments_controller.rb [hunk 3]
```diff
@@ -33,10 +34,31 @@ def update
...
+    end
+    respond_to do |format|
+      format.html do
+        flash[:danger] = flash
+        redirect_to posts_url
+      end
+      format.js
+    end
+  end
+
```
Comment (imogenmisso) [inline app/controllers/comments_controller.rb]: 🌟🌟

File: app/controllers/posts_controller.rb [hunk 4]
```diff
@@ -1,4 +1,5 @@
 # frozen_string_literal: true
+require 'pry'
```
Comment (imogenmisso) [inline app/controllers/posts_controller.rb]: Can remove

File: app/models/like.rb [hunk 5]
```diff
@@ -0,0 +1,4 @@
+class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :likeable, polymorphic: true
```
Comment (imogenmisso) [inline app/models/like.rb]: LOVE

**Comment by runticle** [comment]
⭐️

**Pull request merged**